Lab01 and Lab02 finished

JakubJagodzinski opened the pull request.

Commit:
Fixed drawing set position and place where it is being drawed

Commit:
MyAdapter renamed to AdapterToDrawPanelControllerToJob2DDriver

Commit:
AdapterToDrawPanelControllerToJob2DDriver toString method fixed

Commit:
Added LineDrawerAdapter

Commit:
Added AdapterFromJob2DDriverToAbstractDriver

Commit:
Fixed bug with setPosition method call

Commit:
Removed unnecessary inheritance

Commit:
Added more figure drawing options

Commit:
Added dot line drawer

Commit:
Created DriverCommand interface and command classes

Commit:
Created specialized factories with FigureFactory base class

Commit:
Added 2 new figures drawing options to Patterns

Commit:
Added 2 new figures drawing options

kowallus commented on src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java:
```diff
@@ -19,35 +21,42 @@ public class TestJobs2dPatterns {
 		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
 				DriverFeature.getDriverManager());
 
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
+		application.addTest("Figure Joe 2", selectTestFigureOptionListener);
+		application.addTest("Figure Jane 1", selectTestFigureOptionListener);
+		application.addTest("Figure Rectangle 1", selectTestFigureOptionListener);
+		application.addTest("Figure Triangle 1", selectTestFigureOptionListener);
```
Figure enums should be used also here. Changing only one leads to an error (inconsistency).

kowallus commented on src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDrawerAdapter.java:
```diff
@@ -0,0 +1,40 @@
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class LineDrawerAdapter extends DrawPanelController implements Job2dDriver {
```
Inheritance is not used (no super drawLine calls). Should be removed!

kowallus reviewed (changes requested):
Please fix issues pointed out in reviews and fix the diagram (the inheritance between adapter and DrawPanelController is incorrect - it should be aggregation).

Commit:
Moved Figure enum to another file

Commit:
Replaced strings with relevant Figure enum values

Commit:
Removed unused imports

Commit:
Small fixes:
- Removed unnecessary semicolon in Job2dDriverTest.java
- Replaced function with lambda in TestJobs2dPatterns.java
- Removed unnecessary inheritance in LineDrawerAdapter.java

Commit:
Small fixes suggested by GIT to avoid commit warnings

Commit:
Merge branch 'fix/small_fixes'
